.github/workflows/test.yml

```diff
@@ -335,25 +336,63 @@ jobs:
         with:
           name: verdaccio-log-${{ matrix.template }}
           path: packages/pwa-kit-create-app/local-npm-repo/verdaccio-windows-${{ matrix.template }}.log
-  lighthouse:
+
+  # NOTE: Commenting out this lighthouse test because it's run against a _local_ server that skews the performance metrics to be better than they really are.
+  # TODO: perhaps run lighthouse in another workflow (e2e one) that deploys to MRT and test against that.
+  # lighthouse:
+  #   needs: changelog-check
+  #   strategy:
+  #     fail-fast: false
+  #   runs-on: ubuntu-latest
+  #   env:
+  #     PROJECT_DIR: generated-retail-react-app-test-project
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v4
+
+  #     - name: Setup Ubuntu Machine
+  #       uses: "./.github/actions/setup_ubuntu"
+  #       with:
+  #         node: ${{ env.MRT_NODE }}
+
+  #     - name: Generate retail-react-app-test-project
+  #       run: |-
+  #         node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir ${{ env.PROJECT_DIR }}
+  #       env:
+  #         GENERATOR_PRESET: retail-react-app-test-project
+  #       timeout-minutes: 8
+
+  #     - name: Run Lighthouse CI on the generated project
+  #       uses: "./.github/actions/lighthouse_ci"
+  #       with:
+  #         project_dir: ${{ env.PROJECT_DIR }}
+  #         config_filename: lighthouserc.retail-react-app.js
+
+  bundle-sizes:
     needs: changelog-check
     strategy:
       fail-fast: false
-      matrix:
-        node: [22] # Should match node version supported by MRT.
     runs-on: ubuntu-latest
+    env:
+      PROJECT_DIR: generated-retail-react-app-test-project
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Setup Node
-        uses: actions/setup-node@v4
-        with:
-          node-version: ${{ matrix.node }}
-          cache: npm
-
       - name: Setup Ubuntu Machine
         uses: "./.github/actions/setup_ubuntu"
+        with:
+          node: ${{ env.MRT_NODE }}
+
+      - name: Generate retail-react-app-test-project
+        run: |-
+          node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir ${{ env.PROJECT_DIR }}
+        env:
+          GENERATOR_PRESET: retail-react-app-test-project
+        timeout-minutes: 8
 
-      # - name: Run Lighthouse CI on the PWA
-      #   uses: "./.github/actions/lighthouse_ci"
+      - name: Check and report bundle sizes
+        uses: "./.github/actions/bundle_size_test"
+        with:
+          cwd: ${{ env.PROJECT_DIR }}
+          config: '{"build/main.js": "10kB", "build/vendor.js": "390kB"}'
```

Comment:
What is the reasoning of defining the file sizes in the GitHub workflow rather than define these values in the package.json file?

Reply:
Since we no longer have retail-react-app package, we don't have a file where we can put those config values. The typescript-minimal template is not a good place either, because by default, it's minimal and does not have the same content as retail-react-app.

Alternatively, we can have a config file and put it inside the `actions/bundle_size_test` folder. But I ran into a bug: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/f096f6fb7ac4987fdb2708880c5ab7bba8a1701a/.github/actions/bundle_size_test/action.yml#L18-L20